Add JS sample for voice assistants (#1218)
Azure-Samples/cognitive-services-speech-sdk: samples/js/browser/DialogServiceConnector/README.md

> # JavaScript voice assistants sample for web browsers  

Reviewer: ```suggestion
# JavaScript voice assistant sample for web browsers  
```

> This sample demonstrates how to connect to and interact with your voice assistant using the Speech SDK for JavaScript on a web browser, like Microsoft Edge, or Chrome.

Reviewer: ```suggestion
This sample shows how to connect to and interact with your voice assistant using the Speech SDK for JavaScript on a web browser, like Microsoft Edge, or Chrome.
```

> See [this page](https://github.com/microsoft/cognitive-services-speech-sdk-js) for introductory information on the Speech SDK for JavaScript.

Reviewer 2: Are we allowing to put page tags in URLs here? If so, this should probably be https://github.com/microsoft/cognitive-services-speech-sdk-js#readme

> See [this page](https://github.com/microsoft/cognitive-services-speech-sdk-js) for introductory information on the Speech SDK for JavaScript.

Reviewer: ```suggestion
See [this information on the Speech SDK for JavaScript](https://github.com/microsoft/cognitive-services-speech-sdk-js).
```

## Prerequisites

* A subscription key for the Speech service. See [Try the speech service for free](https://docs.microsoft.com/azure/cognitive-services/speech-service/get-started).
* A voice assistant registered to Direct Line Speech. See [this tutorial](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/tutorial-voice-enable-your-bot-speech-sdk) for a walkthrough.
* A PC or Mac, with a working microphone.
* A text editor.

## Build the sample

* **By downloading the Microsoft Cognitive Services Speech SDK when building this sample, you acknowledge its license, see [Speech SDK license agreement](https://docs.microsoft.com/azure/cognitive-services/speech-service/license).**
* [Download the sample code to your development PC.](/README.md#get-the-samples)

## Run the sample

* In case you are running the sample from your local computer, open `dialogServiceConnectorSample.html` from the location where you have downloaded this sample with a JavaScript capable browser.
* Use the input fields to set your subscription key and service region.

> * Click 'Connect' to establish a connection with your credentials

Reviewer: ```suggestion
* Click **Connect** to establish a connection with your credentials
```

> * Click 'Start listening' and speak a phrase to your bot

Reviewer: ```suggestion
* Click **Start listening** and speak a phrase to your bot.
```
